person.rb

```diff
@@ -1,12 +1,17 @@
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :rentals
```

Comment:
- [ ] Just as pointed as regards :rentals in book.rb file, same applies to this, kindly fix it up.